apps/lockfile-explorer-web/src/containers/LogoPanel/index.tsx

```diff
@@ -15,14 +15,15 @@ export const LogoPanel = (): JSX.Element => {
         <div className={styles.Icon}>
           <img className={styles.Image} src={require('./lockfile-explorer-icon.svg')} />
         </div>
-        <div className={styles.Title1}>
-          <img className={styles.Image} src={require('./lockfile-explorer-title-1.svg')} />
+        <div>
+          <div className={styles.Title1}>
+            <img className={styles.Image} src={require('./lockfile-explorer-title-1.svg')} />
+          </div>
+          <div className={styles.Title2}>
+            <img className={styles.Image} src={require('./lockfile-explorer-title-2.svg')} />
+          </div>
+          <div className={styles.Detail}>{appPackageVersion}&nbsp;rushstack.io</div>
```

Comment:
The old `LogoPanel` looked like this:
![image](https://user-images.githubusercontent.com/4673363/202324025-1bf3cc7f-d390-43a8-9b12-3d6a86215725.png)

In this PR, the layout is changed to be:
![image](https://user-images.githubusercontent.com/4673363/202324208-657fce1b-6d7b-40d9-b8af-364eed2be3a4.png)

The `0.0.3 rushstack.io` seems a bit confusing, since `0.0.3` is the version of Lockfile Explorer, not Rush Stack.

Also in the original design, I had envisioned that clicking on the words/icon would take you to the NPM page (or https://lfx.rushstack.io ?), whereas clicking on `rushstack.io` would take you to the other website https://rushstack.io

What do you think?  Another consideration is that the Projects/Packages box is the centerpiece of the UI, so we want to provide as much vertical scroll area as possible.

Reply:
That makes sense, I've moved the version next to the lockfile explorer text logo to make it a bit more clear, also I moved the logo to above the bookmarks sidebar as that one is less important than the left panel. The appropriate links have also been added. I moved it away from having it's own column because i felt that it was taking up a lot of horizontal real-estate for a pretty small logo, this way I think gives the app overall more room:

<img width="327" alt="image" src="https://user-images.githubusercontent.com/8737494/202331600-59066903-cb04-48fa-8893-d7849e140023.png">